### PR TITLE
Mage_Admin - DOC block update

### PR DESCRIPTION
**Pull request by sreichel**
- doc blocks added/fixed
- PSR2 fixes (whitespaces, linebreaks, ...)

### CODE REVIEW

File: app/code/core/Mage/Admin/Model/Acl/Assert/Ip.php [hunk 1]
```diff
@@ -37,18 +37,24 @@ class Mage_Admin_Model_Acl_Assert_Ip implements Zend_Acl_Assert_Interface
...
+        Mage_Admin_Model_Acl $acl,
+        Mage_Admin_Model_Acl_Role $role = null,
+        Mage_Admin_Model_Acl_Resource $resource = null,
+        $privilege = null
+    ) {
         return $this->_isCleanIP(Mage::helper('core/http')->getRemoteAddr());
     }
 
+    /**
+     * @param bool $ip
```
Comment (tmotyl) [inline app/code/core/Mage/Admin/Model/Acl/Assert/Ip.php:56]: this should be string as Mage::helper('core/http')->getRemoteAddr() returns string

Comment (tmotyl) [reply app/code/core/Mage/Admin/Model/Acl/Assert/Ip.php:56]: maybe return should be bool

File: app/code/core/Mage/Admin/Model/Acl/Assert/Time.php [hunk 2]
```diff
@@ -27,28 +27,34 @@
...
+        Mage_Admin_Model_Acl $acl,
+        Mage_Admin_Model_Acl_Role $role = null,
+        Mage_Admin_Model_Acl_Resource $resource = null,
+        $privilege = null
+    ) {
         return $this->_isCleanTime(time());
     }
 
+    /**
+     * @param bool $time
```
Comment (tmotyl) [inline app/code/core/Mage/Admin/Model/Acl/Assert/Time.php:56]: $time is not bool, but return type yes

File: app/code/core/Mage/Admin/Model/Acl/Assert/Ip.php (hunk 1 above)
Comment (sreichel) [reply app/code/core/Mage/Admin/Model/Acl/Assert/Ip.php:56]: Mhhh ... this methode is empty ....  How abaout ...

- `@param`: `Mage::helper('core/http')->getRemoteAddr()` returns `string` or `false` ... 
- `@return void`

```
    protected function _isCleanIP($ip)
    {
        // ...
    }
```

File: app/code/core/Mage/Admin/Model/Acl/Assert/Time.php (hunk 2 above)
Comment (sreichel) [reply app/code/core/Mage/Admin/Model/Acl/Assert/Time.php:56]: Similar to above ....

File: app/code/core/Mage/Admin/Model/Acl/Assert/Ip.php (hunk 1 above)
Comment (tmotyl) [reply app/code/core/Mage/Admin/Model/Acl/Assert/Ip.php:56]: ok, seems the classes are not used in the core anyway

**Pull request merged**